index.html

```diff
@@ -32,8 +34,15 @@ <h1>Please choose the ingredients you wish to have included:</h1>
             <button type="button"
                 class="text-white bg-gradient-to-br from-purple-600 to-blue-500 hover:bg-gradient-to-bl focus:ring-4 focus:outline-none focus:ring-blue-300 dark:focus:ring-blue-800 font-medium rounded-lg text-sm px-5 py-2.5 text-center me-2 mb-2">Random
                 Search</button>
+            
         </section>
-
+       
+        <section><!--recipe save button-->
```

Comment:
Since the recipes are rendering on the recipe page, I think we should move this to that page instead so we can link it with the specific recipe they want to save